### ISSUE TITLE
[Blocker] Relation selector are still Required in the AntForm even when the BluePrint is not

### ISSUE THREAD

**Issue by Fen747** [bluelibs/bluelibs#206]
**Describe the bug**
My relations all look slike : 
```typescript
    relation({
      id: "requiredConsumables",
      to: "Consumables",
      isMany: true,
      representedBy: "fullName",
      ui: {
        label: "Required Consumables",
      },
      field: field({
        id: "requiredConsumablesIds",
        ui: {
          list: false,
        },
        type: field.types.OBJECT_ID,
        isArray: true,
        isRequired: false,
        defaultValue: [],
      }),
    }),
```

However, in the generated form, the selector ALWAYS require some value, even though it shouldn't (I've set the blueprint's field to `isRequired: false` and `defaultValue: []`).

<img width="732" alt="Screen Shot 2022-02-14 at 10 26 24" src="https://user-images.githubusercontent.com/13072245/153836526-50c0df3a-f9f5-41ee-bc10-158280ed1dd1.png">

**Comment by y-moufakkr** [comment]
Hello @Fen747, try adding `isRequired: false ` to the relation object, currently you are having it on the field object only, which makes the relation required

**Comment by theodorDiaconu** [comment]
Should we display a warning when both relation and field requiredness mismatch?

**Comment by Fen747** [comment]
Sure we should. It's kinda perturbing to know where I need to put some info, etc. For example, I want to keep the auto generated `XXXid` name, but have it be an empty array by default, should I set the `field` option in the relation just to have this defaultValue ? Or do I need to add `defautlValue` in the relation directly ?

IMHO, the `field` option should not exists, we just set everything in the `relation`, even default value, and just have an optional `fieldName` string.
